frontend_next/app/(root)/layout.tsx

```diff
@@ -0,0 +1,29 @@
+import type { Metadata } from "next";
+import type { ReactNode } from "react";
+import { Toaster } from "@/components/ui/toaster";
+
+import "../globals.css";
```

Comment:
```suggestion
import "../globals.css";
```

In your css you're using an @import to import your font once the page loads. Next let's you do this more efficiently by optimizing and including the font in the initial package sent to the client:

```js
import { Poppins } from 'next/font/google'
 
// If loading a variable font, you don't need to specify the font weight
const poppins = Poppins({ subsets: ['latin'] ,weight: ['400', '500','600','700'],})
 
export default function MyExampleApp() {
  return (
    <main className={Poppins.className}>
  
    </main>
  )
}
```

If you attach the poppins.classname to the body of your root layout and remove all other references to the fonts in your CSS, everything will automatically use Next's optimized fonts.

Reply:
https://nextjs.org/docs/app/building-your-application/optimizing/fonts